### ISSUE TITLE
ct<leader> or cf<leader> doesn't work.  Need ct<leader><leader>

### ISSUE THREAD

**Issue by gerardroche** [NeoVintageous/NeoVintageous#81]
<a href="https://github.com/johnathanz"><img src="https://avatars3.githubusercontent.com/u/1593039?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [johnathanz](https://github.com/johnathanz)**
_Tuesday Sep 30, 2014 at 14:43 GMT_
_Originally opened as https://github.com/guillermooo/Vintageous/issues/741_

----

Great work on Vintageous btw, it was the straw that converted me to ST3 :+1: 

I'm trying to carry over the concept of <leader> to Vintageous purely through key mappings.  I use "," as <leader> and have a number of keys mapped as

[",", "t"]  or [",", "i"]

unfortunately this causes Vintageous to wait for another character if "," is the last key in a command.

Say deleting a line up to & including ","

When I press "cf," the comma isn't registered, I have to press comma again for it to work.  I suspect Vim has a keypress delay that's used to gauge whether it's a <leader><shortcut>, or a command that ends on the <leader> key.

Is there anyway this could be fixed in Vintageous?

**Comment by gerardroche** [comment]
<a href="https://github.com/gerardroche"><img src="https://avatars0.githubusercontent.com/u/44148?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [gerardroche](https://github.com/gerardroche)**
_Friday Dec 19, 2014 at 16:52 GMT_

----

:+1:

**Comment by gerardroche** [comment]
@johnathanz 1.3.0 adds the ability to create `.vintageousrc` mappings using a a leader keay anywhere in the mapping. https://github.com/NeoVintageous/NeoVintageous/blob/master/CHANGELOG.md#130---2017-07-21

```
let mapleader=,

" Visually select all content
nnoremap cf<leader> ggvG
```

Currently you can only map to commands that Neovintageous understands and can't yet map to ex mode commands `nnoremap cf<leader> :w`. This will be implemented hopefully by v1.5.

**Comment by gerardroche** [comment]
@johnathanz This has been fixed in the latest 1.4.0 release (NeoVintageous is a fork of the discontinued Vintageous plugin):

The issue with Sublime Text mappings is they circumvent NeoVintageous. NeoVintageos remaps keys, but when you create a mapping like `[",", "t"]` and then press `,`, ST sees that key is ambiguous so it has to wait for the key press.

The way to fix this is to create mappings in your `vintageousrc` file so that the key presses are registered with Neovintageous.

> Mapping to Command-line mode commands is supported for basic use-cases: >
>     :command<CR>
>
> Mapping to Sublime Text commands is supported for basic use-cases. The command
> must start an Uppercase letter to avoid confusion with built-in Command-line
> mode commands. The command is converted to snake_case. For example to map to
> the Sublime Text command "toggle_side_bar": >
>     :ToggleSideBar<CR>
>
> Here is an example |vintageousrc| file: >
>
>     " The character " (the double quote mark) starts a comment
>
>     let mapleader=,
>
>     " Enter command line mode.
>     nnoremap <space> :
>
>     " Toggle the side bar.
>     nnoremap <leader>d :ToggleSideBar<CR>

See the [`:help neovintageous`](https://github.com/NeoVintageous/NeoVintageous/blob/master/res/doc/neovintageous.txt) command for information.

Also see the forum post on latest release: [NeoVintageous 1.4.0 released zap KAPOW](https://forum.sublimetext.com/t/neovintageous-1-4-0-released-zap-kapow/33172).
